src/styles/index.scss

```diff
@@ -1,3 +1,169 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  border: none;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+
+  color: #616070;
+
+  display: flex;
+  justify-content: center;
+}
+
+.product-card {
+  &__wrapper {
+    padding: 32px 16px 16px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+
+    max-width: 200px;
+    width: 100%;
+
+    background-color: #ffffff;
+  }
+
+  &__header {
+    max-width: 160px;
+    width: 100%;
+
+    height: 134px;
+
+    &:not(:last-child) {
+      margin-bottom: 40px;
+    }
+  }
+
+  &__img {
+    width: 100%;
+    height: 100%;
+
+    object-position: center;
+  }
+
+  &__main {
+    &:not(:last-child) {
+      margin-bottom: 16px;
+    }
+  }
+
+  &__footer {
+    max-width: 100%;
+    width: 100%;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    padding: 12px;
+    max-width: 100%;
+    width: 100%;
+
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    color: #ffffff;
+
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    background-color: #00acdc;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    transition: 0.3s background-color;
+
+    &:hover {
+      background-color: transparent;
+      border-color: #00acdc;
+      color: #00acdc;
```

Comment:
You also change `border-color` and `color` so apply transition for them as well

Reply:
ok